assets/src/util/fullstory.tsx

```diff
@@ -0,0 +1,20 @@
+import { getDatasetValue } from "Util/dataset";
+import * as FullStory from "@fullstory/browser";
+import { isRealScreen } from "./util";
+
+/**
+ * Initializes Sentry if the DSN is defined AND this client is running on
+ * a real production screen AND the URL does not contain the disable_sentry param.
+ */
+const initFullstory = () => {
+  const screenplayFullstoryOrgId = getDatasetValue("screenplayFullstoryOrgId");
+
+  if (screenplayFullstoryOrgId && !isRealScreen()) {
```

Comment:
What is the purpose of the `!isRealScreen()` check? In all situations where `screenplayFullstoryOrgId` is set, won't it not be a real screen?

Reply:
Mostly to make me feel more confident that this will not initialize for real screens. It very well may be overkill because we only get to this code from the simulation endpoint.